src/adera/machines/links.h

```diff
@@ -47,11 +51,27 @@ PortEntry const gc_rollOut          { gc_ntSigFloat, 3, gc_sigOut };
 
 namespace ports_magicrocket
 {
-using osp::link::PortEntry;
-using osp::link::gc_ntSigFloat;
-using osp::link::gc_sigIn;
-
 PortEntry const gc_throttleIn       { gc_ntSigFloat, 0, gc_sigIn };
+PortEntry const gc_multiplierIn     { gc_ntSigFloat, 1, gc_sigIn };
 }
 
+namespace ports_rcsdriver
+{
+PortEntry const gc_posXIn           { gc_ntSigFloat, 0, gc_sigIn };

```

Comment:
Can't these be constexpr?

If not constexpr, perhaps constinit?

Reply:
gc_ntSigFloat is dynamically initialized. I'm having a difficult time telling whether there is any ordering issues here, address sanitizer doesn't seem to mind though.